src/pages/in/embed/_id/list.vue

```diff
@@ -109,7 +131,7 @@ export default {
     ] = await Promise.all(promises);
     let title = '';
     if (html) {
-      const match = html.match('<title>(.*?)</title>');
+      const match = html.match(/<title(?: [a-zA-Z0-9-="']+)?>(.*?)<\/title>/);
```

Comment:
1. does not match the case of multiple spaces after `<title`? e.g. `<title  abcd>...` (with 2 spaces)
2. does not match the case where there is space in quotation, e.g. `<title some-tag="This is sparta">...`

Maybe simply `<title(?:[^>]*)?>(.*?)<\/title>`?

Reply:
No, as it matches until next `>`, which will not escape from the `title` tag